http/oidc/src/main/java/org/wildfly/security/http/oidc/ServerRequest.java

```diff
@@ -274,4 +279,40 @@ public static AccessAndIDTokenResponse getBearerToken(OidcClientConfiguration oi
         }
         return tokenResponse;
     }
+
+    public static String getRequestUri(String request, OidcClientConfiguration deployment) throws OidcException {
+        HttpPost parRequest = new HttpPost(deployment.getPushedAuthorizationRequestEndpoint());
```

Comment:
Can getPushedAuthorizationRequestEndpoint() return null?

Reply:
I have added an error message for when this is null